defaults.yaml

```diff
@@ -11,6 +11,7 @@ conf:
   minion_2_name: k8s-minion-2
   minion_3_name: k8s-minion-3
   kubernetes_nodes_base_box_id: "bento/centos-7.4"
+  ansible_debug: off
```

Comment:
What if we call this debug_output instead of ansible_debug? We could reuse this for other purposes as well.

WDYT?

Reply:
agreed and changed ! (also in Vagrantfile)